gitops/azure-devops/build.sh

```diff
@@ -1,24 +1,30 @@
+#!/bin/bash
+
 function init() {
     cp -r * $HOME/
     cd $HOME
 
+    echo "CHECKING MANIFEST REPO URL"
+    if [[ -z "$AKS_MANIFEST_REPO" ]]; then
+        echo 'MANIFEST REPO URL not specified in variable $MANIFEST REPO URL'
+        exit 1
+    fi
+
     echo "CHECKING GIT HOST"
     if [[ "$GIT_HOST" == "github" ]]; then
-        git_dest_repo="https://github.com/$AKS_MANIFEST_REPO"
+        git_dest_repo="$AKS_MANIFEST_REPO"
         git_type=$GIT_HOST
     elif [[ "$GIT_HOST" == "azure" ]]; then
-        git_dest_repo="https://dev.azure.com/$AKS_MANIFEST_REPO" # For repos that reside in Azure Devops, the AKS_MANIFEST_REPO should be formatted like "user_account/project_name/_git/repo_name"
+        git_dest_repo="$AKS_MANIFEST_REPO" # For repos that reside in Azure Devops, the AKS_MANIFEST_REPO should be formatted like "user_account/project_name/_git/repo_name"
```

Comment:
Why are we removing github.com / dev.azure.com?

Reply:
@samiyaakhtar Previously we wanted the user to have to parse off the organization and repo name from the URL. Now we just ask for an absolute URL which is simpler. Agree?

I will delete this comment here that says how the AKS_MANIFEST_REPO should be formatted. 

I added new commentary about how this should look in the [instructions](https://github.com/Microsoft/bedrock/blob/292b57c5868892f2b97614f3bf9901006ae9eb5c/gitops/azure-devops/README.md#azure-pipelines-build-yaml)

Reply:
Has it been tested? Since the rest of the script accepts it in that form, not sure if more changes will be necessary to accommodate for this change

Reply:
@samiyaakhtar Yes. All these build.sh changes are based on testing of our instructions and finding several issues.

You're right to point out that we don't need $git_type anymore. We were only using $git_type in when git pushing. Now that we use the absolute git URL we don't care about the git type anymore. 

I can remove the entire `git check` method and remove instructions to have to set it in the pipeline variables I that makes sense to you.